src/SprykerEco/Yves/Braintree/Theme/default/components/molecules/braintree-pay-pal-express/braintree-pay-pal-express.ts

```diff
@@ -66,29 +82,29 @@ export default class BraintreePayPalExpress extends Component {
             style: {
                 size: 'responsive'
             },
-            payment: () => this.onPaymentHandler(paypalCheckoutInstance),
-            onAuthorize: (data, actions) => this.onAuthorizeHandler(data, actions, paypalCheckoutInstance),
-            onError: (error) => {
+            payment: () => this.onPaymentHandler(checkoutInstance),
+            onAuthorize: (data, actions) => this.onAuthorizeHandler(data, actions, checkoutInstance),
+            onError: error => {
                 console.error('PayPal checkout button render error!', error);
             }
         }, `.${ this.name }`);
-    };
+    }
 
-    protected onPaymentHandler(paypalCheckoutInstance: IPaypalCheckoutInstance): Function {
-        return paypalCheckoutInstance.createPayment({
+    protected onPaymentHandler(checkoutInstance: PaypalCheckoutInstance): Function {
+        return checkoutInstance.createPayment({
             flow: 'checkout',
             intent: 'authorize',
             amount: this.braintreeData.amount,
             currency: this.braintreeData.currency,
             enableShippingAddress: true,
             shippingAddressEditable: true
         });
-    };
+    }
 
-    protected onAuthorizeHandler(data, actions, paypalCheckoutInstance: IPaypalCheckoutInstance): Function {
-        return paypalCheckoutInstance.tokenizePayment(data).then((payload) => {
-            payload['amount'] = this.braintreeData.amount;
-            payload['currency'] = this.braintreeData.currency;
+    protected onAuthorizeHandler(data: object, actions: object, checkoutInstance: PaypalCheckoutInstance): Function {
```

Comment:
Suggest, changing object type to the appropriate interface

Reply:
https://www.npmjs.com/package/@types/braintree-web

Reply:
The same with proposed types, it'a appropriate partly.